ubuntu-deploy/index.md

```diff
@@ -0,0 +1,414 @@
+
+Your application should now be accessible at `http://your_server_ip:8055`.
+
+![Directus application accessed with the server ip at port 8055][image-2]
+
+:::info Error Debugging
+
+If you encounter any error e.g `SQLITE_CANTOPEN: unable to open database file`, it is probably due to permission issues. You can try running this command to fix it:
+
+```bash
+sudo chown -R $USER:$USER /path/to/your/directus/folder
+
+```
+
+More info about this error in this [issue][issue]
+
+:::
+
+## Running the Docker Container as a Background Service
+
+Running your application like above is fine, but it will stop running when you close the terminal.
+
+To ensure your application runs in the background and restarts automatically, you can create a systemd service.
+
+:::info What is Systemd?
+
+Systemd is a system and service manager for Linux operating systems. It provides a standard process for controlling the startup, management, and monitoring of services and applications. It is usually defined by a configuration file usually ending with the _.service_ extension.
+
+:::
+
+### Create a Systemd Service File
+
+Create a file named _directus.service_ in the _/etc/systemd/system/_ directory:
+
+```bash
+sudo vi /etc/systemd/system/directus.service
+```
+
+Add the following content, updating the `WorkingDirectory` to your Directus project directory containing the _docker-compose.yml_ file:
+
+```plaintext
+[Unit]
+Description=Directus Docker Service
+Requires=docker.service
+After=docker.service
+
+
+[Service]
+Restart=always
+WorkingDirectory=/path/to/your/directory-containing-docker-compose
+ExecStart=/usr/bin/docker-compose up
+ExecStop=/usr/bin/docker-compose down
+
+
+[Install]
+WantedBy=multi-user.target
+```
+
+:::info
+
+You can get the full path to you directory by running the command `pwd` in the project directory on your server and copying the output.
+
+![print working directory(pwd) command][image-10]
+
+:::
+
+Save the file and exit the editor.
+
+Let's step through the service file:
+
+- [Unit] Section:
+
+  - Description: Description of the systemd service.
+  - Requires: Dependency on the Docker service.
+  - After: Starts after the Docker service.
+
+- [Service] Section:
+
+  - Restart=always: The service restarts automatically on exit.
+  - WorkingDirectory: Path to the directory containing the docker-compose.yml file.
+  - ExecStart: Command to start Docker containers.
+  - ExecStop: Command to stop Docker containers.
+
+- [Install] Section:
+  - WantedBy=multi-user.target: Service enabled on reaching the multi-user state.
+
+### Enable and Start the Service
+
+Enable the service to start on boot, and then start the service:
+
+```bash
+sudo systemctl enable directus.service
+sudo systemctl start directus.service
+```
+
+Now, your dockerized Directus application is running as a background service and will automatically restart in case of failures or system reboots.
```

Comment:
I'd love you to add an explicit note if this works across system restarts as well